Kernel/Syscalls/setkeymap.cpp

```diff
@@ -27,6 +27,8 @@
 #include <Kernel/Devices/KeyboardDevice.h>
 #include <Kernel/Process.h>
 
+#define CHAR_MAP_NAME_SIZE 50
```

Comment:
No need for this to be a macro, instead just use a function-local `constexpr size_t`